target_postgres/db_sync.py

```diff
@@ -329,7 +329,7 @@ def table_name(self, stream_name, is_temporary=False, without_schema=False):
         if without_schema:
             return f'"{pg_table_name.lower()}"'
 
-        return f'{self.schema_name}."{pg_table_name.lower()}"'
+        return f'"{self.schema_name}"."{pg_table_name.lower()}"'
```

Comment:
I'm fairly sure that this won't work.  Double quotes are used for identifiers, but single quotes are used for strings (which is what you want here).